docs/Formula-Cookbook.md

```diff
@@ -776,6 +776,14 @@ brew search --fink foo
 
 Some software requires a Fortran compiler. This can be declared by adding `depends_on "gcc"` to a formula.
 
+## MPI
+
+Formula requiring MPI should use [OpenMPI](https://www.open-mpi.org/) by adding `depends_on "open-mpi" to the formula, rather than [MPICH](https://www.mpich.org/). These packages have conflicts and provide the same standardized interfaces. Choosing a default implementation and requiring it to be adopted allows software to link against multiple libraries that rely on MPI without creating un-anticipated incompatibilities due to differing MPI runtimes.
```

Comment:
Missing backtick behind "open-mpi"

Reply:
957f382 fixes this. I'll rebase & force push once I figure out what to do about Mike's comment.